pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.lfenergy.compas</groupId>
+  <artifactId>compas-scl-xsd</artifactId>
+  <version>0.1-SNAPSHOT</version>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <version>1.6.0</version>
+        <executions>
+          <execution>
+            <id>get-1.7-xsd</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
+              <uri>https://assets.iec.ch/public/tc57/IEC_61850-6.2003.SCL.1.7.full.zip</uri>
+              <outputDirectory>${project.basedir}/src/xsd/SCL1.7</outputDirectory>
+              <unpack>true</unpack>
+              <md5>2f912148b013350e6d09ffce64866d9a</md5>
+            </configuration>
+          </execution>
+          <execution>
+            <id>get-2007B-xsd</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
+              <uri>https://assets.iec.ch/public/tc57/IEC_61850-6.2009.SCL.2007B.full.zip</uri>
+              <outputDirectory>${project.basedir}/src/xsd/SCL2007B</outputDirectory>
+              <unpack>true</unpack>
+              <md5>13dc10768c4ac8d58fcf70491eec3f3d</md5>
+            </configuration>
+          </execution>
+          <execution>
+            <id>get-2007B4-xsd</id>
```

Comment:
What is the difference between all 3 sets of XSDs and why do we need al 3 of them?

Reply:
The 3 sets matches with all available main version to follow. As SCT tool, we need to be able to read / write old main versions of SCL files. So, we should be able to understand Ed1, Ed2 and Ed2.1 SCL files.
We the standard will publish a new one, we should add the support for it (without remove support of older)

Maybe the Ed3 will come with possibility to not be compatible with Ed1 devices. But it's not sure.

Reply:
Got it! Thanks!